contracts/test/GenericMock.sol

```diff
@@ -62,10 +63,21 @@ contract GenericMock {
         mockValues[_func].set = true;
     }
 
+    /*
+     * @dev Set a mock address value for a function
+     * @param _func Function selector (bytes4(keccak256(FUNCTION_SIGNATURE)))
+     * @param _value Mock address value
+     */
+    function setMockAddress(bytes4 _func, address _value) external {
+        mockValues[_func].valueType = MockValueType.Address;
+        mockValues[_func].addressValue = _value;
+        mockValues[_func].set = true;
+    }
+
     /*
      * @dev Return mock value for a functione
      */
-    function() public {
+    function() public payable {
```

Comment:
Allows this mock contract to receive ETH (relevant for testing `deposit()` in JobsManager which forwards ETH to the Minter)